setup.py

```diff
@@ -119,7 +119,7 @@ class StyleCommand(distutils_cmd.Command):
         'isort '
         # These two lines makes isort compatible with black.
         '--multi-line=3 --trailing-comma --force-grid-wrap=0 '
-        f'--use-parentheses --line-width={__LINE_WIDTH} -rc'
+        f'--use-parentheses --line-width={__LINE_WIDTH} -rc -ws'
```

Comment:
Why are we adding this?

Reply:
the "-ws" was breaking isort on Sergei's and mine machines. Anyway, it tells isort to ignore some whitespaces, which makes it less painful

Reply:
what is that option responsible for?

Reply:
it resolves the conflict between `isort` and `black`. These two have different opinions regarding empty lines between `import` statements. `-ws` makes `isort` to relax about it